client/components/EventItem/index.jsx

```diff
@@ -19,7 +19,7 @@ export const EventItem = ({ event, onClick }) => {
             <div className="line">
                 <span className="keyword">{event.name}</span>
                 <span className="item-heading">{event.definition_short}</span>
-                <time title={time}>{time}</time>
```

Comment:
With BEM, when we define an element state, we use `--` instead of `__`.
I made this error few times already, you can blame me when you find one in the project ;)

http://getbem.com/introduction/